### ISSUE TITLE
autosummary does not show info for instance attributes

### ISSUE THREAD

**Issue by daphtdazz** [sphinx-doc/sphinx#3257]
Subject: autosummary does not show info for instance attributes

### Problem
Autosummary finds instance attributes and puts them in the table, but it does not find an annotation that went with them and put the first line of that in the table

Initially came from a [stack overflow question](http://stackoverflow.com/questions/29902483/how-can-i-get-sphinx-autosummary-to-display-the-docs-for-an-instance-attributes).

#### Procedure to reproduce the problem

Have some code like:

##### test.py
```
class MyClass:
    def __init__(self):
        self.attribute = "An Attribute"
        """This is a cool attribute."""
```

##### test.rst

```
.. autosummary::
    test.MyClass.attribute
```

#### Error logs / results

<table border="1" class="longtable docutils">
<colgroup>
<col width="10%" />
<col width="90%" />
</colgroup>
<tbody valign="top">
<tr class="row-odd"><td><code class="xref py py-obj docutils literal"><span class="pre">test.MyClass.attribute</span></code></td>
<td></td>
</tr>
</tbody>
</table>

#### Expected results

<table border="1" class="longtable docutils">
<colgroup>
<col width="10%" />
<col width="90%" />
</colgroup>
<tbody valign="top">
<tr class="row-odd"><td><code class="xref py py-obj docutils literal"><span class="pre">test.MyClass.attribute</span></code></td>
<td>This is a cool attribute.</td>
</tr>
</tbody>
</table>

Or something.

### Reproducible project / your project

As per the "Procedure to reproduce the problem".

### Environment info
- OS: Linux/Mac (but I expect all of them)
- Python version: 3.5.0
- Sphinx version: 1.3.1

**Comment by shimizukawa** [comment]
Memo: documenting class instance attribute is not implemented in current Sphinx (1.5.1).

**Comment by leotrs** [comment]
This still seems to be the case. I'd love to see it be implemented!
